internal/app/machined/pkg/runtime/v1alpha1/platform/azure/testdata/expected.yaml

```diff
@@ -18,3 +18,9 @@ externalIPs:
     - 1.2.3.4
     - 2603:1020:10:5::34
     - 20.10.5.34
+metadata:
+    platform: azure
+    region: centralus
+    zone: "0"
+    instanceType: Standard_DS2
+    providerId: azure://
```

Comment:
empty provider id?

Reply:
yep, it's missing in the test metadata I found (official Azure sample), I can fill it out as well

Reply:
added fake resourceID to fill it